modules/ting/ting.entities.inc

```diff
@@ -396,17 +396,24 @@ class TingCollection extends DingEntityCollection {
     return $primary_object === NULL ? FALSE : $primary_object->getShortTitle();
   }
 
-  function getEntities() {
+  /**
+   * Get list of entities in this collection.
+   *
+   * @return \Ting\TingObject[]
+   *   List of TingObject instances.
+   */
+  public function getEntities() {
     $this->entities = array();
-    if (isset($this->reply->objects)) {
+    $objects = $this->tingObjectCollection->getObjects();
+    if (!empty($objects)) {
```

Comment:
Why do we need to check for emptyness? I would expect `getObjects()` to return an empty array if one is not available.